src/ir/display.rs

```diff
@@ -37,25 +66,30 @@ impl<'a> Display for FunctionBodyDisplay<'a> {
             ret_tys.join(", ")
         )?;
 
+        self.decorator.before_function_body(f)?;
+
         for (value, value_def) in self.body.values.entries() {
             match value_def {
-                ValueDef::Operator(op, args, tys) if self.verbose => writeln!(
-                    f,
-                    "{}    {} = {} {} # {}",
-                    self.indent,
-                    value,
-                    op,
-                    self.body.arg_pool[*args]
-                        .iter()
-                        .map(|arg| format!("{}", arg))
-                        .collect::<Vec<_>>()
-                        .join(", "),
-                    self.body.type_pool[*tys]
-                        .iter()
-                        .map(|arg| format!("{}", arg))
-                        .collect::<Vec<_>>()
-                        .join(", ")
-                )?,
+                ValueDef::Operator(op, args, tys) if self.verbose => {
+                    write!(
+                        f,
+                        "{}    {} = {} {} # {} ",
+                        self.indent,
+                        value,
+                        op,
+                        self.body.arg_pool[*args]
+                            .iter()
+                            .map(|arg| format!("{}", arg))
+                            .collect::<Vec<_>>()
+                            .join(", "),
+                        self.body.type_pool[*tys]
+                            .iter()
+                            .map(|arg| format!("{}", arg))
+                            .collect::<Vec<_>>()
+                            .join(", "),
+                    )?;
+                    writeln!(f, "")?;
```

Comment:
any particular reason to split out the newline like this?

Reply:
(looking below: ah, is a decorator call missing? or is this for symmetry with the below?)

Reply:
This was a change leftover from refactoring. I've changed it to a `writeln!` call.